testsuite/features/support/constants.rb

```diff
@@ -291,7 +292,9 @@
     'salt_migration_minion' => 'SLE-Product-SLES15-SP5-Pool for x86_64'
   },
   'Uyuni' => {
-    'proxy' => 'openSUSE Leap 15.5 (x86_64)',
+    # WORKAROUND until https://github.com/SUSE/spacewalk/issues/23053 will be done
+    'proxy_containerized' => 'openSUSE Leap 15.5 (x86_64)',
```

Comment:
shouldn't this be like this?
```suggestion
    'proxy_container' => 'openSUSE Leap 15.5 (x86_64)',
```

Reply:
Yes. Nice catch. Thank you, Kostas.

Reply:
I amended it to the correct commit to save some time.